feat: Added page that describes LZ Accelerator (#35)
meshcloud/collie-hub: docs/accelerator/README.md

# Landing Zone Accelerator

## Background

> Collie and its Collie Hub, with all of its available kit modules, are open-source and fully usable for free.
> Anyone can use Collie and the Collie Hub to build and manage their cloud foundation and their landing zones.

Reviewer: ```suggestion
Collie and Collie Hub, with all of its available kit modules, are open-source and fully usable for free.
Anyone can use Collie and Collie Hub to build and manage their cloud foundation and their landing zones.
```

> Collie and the Collie Hub are offered by [meshcloud](https://meshcloud.io/en/), which is a technology startup
> that offers a Cloud Foundation Platform as a Software-as-a-Service.

Reviewer: ```suggestion
Collie and Collie Hub are offered by [meshcloud](https://meshcloud.io/en/), which is a technology startup
that offers a Cloud Foundation Platform as a Software-as-a-Service.
```

Reviewer: d: The part "offers a Cloud Foundation Platform as a Software-as-a-Service" in this context is not really relevant. meshStack does not play a role in the LZ Accelerator. I say we leave this part out to avoid confusion.

To get organizations up-to-speed even quicker, meshcloud offers the
[Azure Landing Zone Accelerator](https://www.meshcloud.io/en/services/landing-zone-accelerator/) as a service to get
an up-and-running Azure Cloud Foundation Architecture within 6 weeks.

## The Azure Landing Zone Accelerator

The Azure Landing Zone Accelerator is **the** way to get your organization to Azure, fast and efficiently.
By doing a number of hands-on workshops with the cloud experts from meshcloud, and the powerful
Collie & Collie Hub technology, organizations can get their Azure Cloud Foundation built incredibly fast.

### Why the Azure Landing Zone Accelerator?

- **Based on open-source IaC tools**: The entire Accelerator uses open-source technology like

>   Collie, and industry-leading IaC tool Terraform, which prevents you from being locked-in to tools.

Reviewer 2: I dont get it, because you dependent to terraform. It allows you to doing IaC over multiple vendors. I would 
 say:  "you are not depent on the tools of the vendors (like cloudformation, biceps and so on".

> - **Based on Microsoft best practices**: The Accelerator is based on
>   [Azure Enterprise Scale](https://github.com/Azure/Enterprise-Scale), which a set
>   of best practices for building a Cloud Foundation, coming directly from Microsoft.

Reviewer: ```suggestion
- **Based on Microsoft best practices**: Many of the best practices used in Landing Zone Accelerator agree with what's included in [Azure Enterprise Scale](https://github.com/Azure/Enterprise-Scale).
```

"based on" is a bit too strong

- **Community-backed**: The Accelerator is backed by the Cloud Foundation Community,
  which regularly contributes new modules.
- **No Powerpoint**: Instead of using Powerpoint slide decks, the Accelerator delivers tangible results
  in the form of usable Infrastructure-as-Code and markdown-based documentation.

### What does it entail?

> In the six weeks, we will work together on setting up all the key Essential capabilities

Reviewer: ```suggestion
In the six weeks, we will work together on setting up all the key Essential (stage 1) capabilities
```

in the [Cloud Foundation Maturity Model](https://cloudfoundation.org/maturity-model/),
which allows you to start using the Azure cloud effectively within your organization.

These capabilities include:

- Document the shared responsibility model of your organization.
- Establish a resource hierarchy in Azure.
- Implement service and location restrictions in Azure.
- Enforce centralized audit logging in Azure.
- Align identity and access management.
- Understand basic cost management in Azure.

> Thanks to Collie, you will be delivered a modular landing zone, which means it
> is very easy to extend your landing zone with further capabilities in the future.

Reviewer: ```suggestion
By design, Collie helps you build up modular landing zones, which means it
is very easy to extend your landing zone with further capabilities in the future.
```

### Add-on packages

Beyond the key Essential capabilities for your Azure Cloud Foundation, there are two add-on packages for those cloud foundation
teams that need it:

**Infrastructure-as-Code Bootcamp**: Don't have Terraform or IaC experience? No problem!

> In a 1-day bootcamp we will get you all the basics of Terraform & Terragrunt.

Reviewer 2: Ambitioned :)
